Python/Product/TestAdapter.Executor/Pytest/PytestTestDiscoverer.cs

```diff
@@ -149,7 +149,12 @@ ITestCaseDiscoverySink discoverySink
             //Note pytest specific arguments go after this separator
             arguments.Add("--");
             arguments.Add("--cache-clear");
-            arguments.Add(String.Format("--rootdir={0}", projSettings.ProjectHome));
+            if (string.IsNullOrEmpty(projSettings.UnitTestRootDir)) {
+                arguments.Add(String.Format("{0}", projSettings.ProjectHome));
+            } else {
+                arguments.Add(String.Format("{0}\\{1}", projSettings.ProjectHome, projSettings.UnitTestRootDir));
```

Comment:
Yeah I accidently changed [another file](https://github.com/microsoft/PTVS/blob/1068d0292c91b0cfe534fa99eb11fef6895a14dd/Python/Product/TestAdapter.Executor/Pytest/PytestConfiguration.cs#L68) that caused the issue. I changed it back to "--rootdir={0}" and it's working now.

Reply:
When we were looking into this, --rootDir didn't seem to do anything (for test discovery anyway). I don't see any duplicate tests in the gifs above?

Basically, the bug is that UnitTestRootDirectory setting in the PythonSettings.json does not work. And it's because --rootDir doesn't filter the directory containing the tests, that's only done by passing the actual path where the tests reside into the pytest command line.

Reply:
(I think this might be because rootDir is automatically set if not specified, and I think it defaults to the current working directory. See https://docs.pytest.org/en/6.2.x/customize.html#finding-the-rootdir)